.ado/variables/vs2019.yml

```diff
@@ -1,6 +1,6 @@
 variables:
   VmImage: windows-2019
   MSBuildVersion: 16.0
-  GoogleTestAdapterPathExpression: '(Get-ChildItem "${env:ProgramFiles(x86)}\Microsoft Visual Studio\2019\Enterprise\Common7\IDE\Extensions\" -Recurse -Include GoogleTestAdapter.Core.dll).Directory.FullName'
+  GoogleTestAdapterPathExpression: '(Get-ChildItem "${env:ProgramFiles(x86)}\Microsoft Visual Studio\2019\Enterprise\Common7\IDE\Extensions\" -directory | Where-Object -filterScript { Test-Path $_\GoogleTestAdapter.Core.dll}).FullName'
```

Comment:
(nit)

Use capital letters for argument names: `-Directory`, `-FilterScript`.

Reply:
Also, I find it peculiar that `Get-ChildItem` does not show the `-Directory` switch in autocomplete for me.
Good to know it exists.

Reply:
Sorry about the casings. Will update... 
PowerShell remains counter-intuitive and opaque to me... I still have to search online every 5 tokens I read or type it...

Reply:
Not a big deal obviously.
Regarding PowerShell's intuitiveness, it gets easier. I promise.